src/encoded/static/components/schema.js

```diff
@@ -33,6 +33,9 @@ const excludedTerms = [
     'boost_values',
     'changelog',
     '@type',
+    'output_type_output_category',
+    'file_format_file_extension',
```

Comment:
Why are these last two excluded?

Reply:
Because those are just static mappings not configured by submitters, and as they’re both root level in the schema, they caused the schema pages to be huge when I first put in these schema pages. With this ticket they would show up collapsed so they could appear without impacting the page too much, but I’m still not clear they offer any useful information to submitters though.